tests/CronJobTest.php

```diff
@@ -87,28 +87,28 @@ public function test_cronjob_api_interaction()
 
     public function runCreationTests()
     {
-        $pi = K8s::container()
-            ->setName('pi')
-            ->setImage('perl', '5.34.0')
-            ->setCommand(['perl',  '-Mbignum=bpi', '-wle', 'print bpi(2000)']);
+        $busybox = K8s::container()
+            ->setName('busybox-exec')
+            ->setImage('public.ecr.aws/docker/library/busybox')
+            ->setCommand(['/bin/sh', '-c', 'sleep 30']);
```

Comment:
A change similar to this may be more useful (and save a little CPU) where we don't actually care about burning CPU for the test and just need a pod to run for some predictable period of time. 

That's what happened and was needed here; shortening the perl computation meant a Job wasn't running long enough for the test to pick it up and validate it in the way it wanted, so I put it back into running with the persistent sleep I saw in another test.